sql/executor.go

```diff
@@ -99,6 +99,11 @@ func (e *Executor) Execute(args driver.Request) (driver.Response, int, error) {
 		}
 		planMaker.setTxn(txn, planMaker.session.Txn.Timestamp.GoTime())
 	}
+	var err error
+	planMaker.evalCtx.Location, err = getTimeZone(planMaker.session)
```

Comment:
In that case we could make it lazy, but a cache would be even better (so we only load each timezone once). And it only loads from `zoneinfo.zip` when the go source tree is present and `/usr/share/zoneinfo` is not (the directory is more likely to be available in production than the zip).